kernels/x86.py

```diff
@@ -24,6 +24,8 @@
 INDIRECT  = r"(jmp|call).*%"
 CALL_RET  = '(call|ret)'
 COND_BR   = 'j[^m][^ ]*'
+TEST_CMP  = r"(test|cmp).?\s"
+LOAD      = r"mov.?\s.*\).*,"
```

Comment:
I suspect this may catch stores as well, but it does not seem to be used by this commit

Reply:
I think the ',' after ')' makes it a load definitely